lib/cryptor.rb

```diff
@@ -1,15 +1,13 @@
 class Cryptor
 
-  attr_reader :message, :key, :date
+  attr_reader :message
 
-  def initialize(message, key = '', date = '') 
-    @message = message
-    @key = key
-    @date = date
+  def initialize(message)
+    @message = message.downcase
```

Comment:
`Cryptor` objects are initialize with a `message`. The `key` and `date` parameters passed in to the child class `initialize` methods are used to initialize the `encrypt/decrypt_key` and `encrypt/decrypt_offset`